code/modules/mob/living/carbon/human/species_types/snail.dm

```diff
@@ -58,5 +58,5 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 
 /obj/item/storage/backpack/snail/Initialize()
-	. = ..()
+	. = .()
```

Comment:
```suggestion
	. = ..()
```
Not sure what you're doing here.
It's like saying 1 is 1 pretty sure you meant to do this

Also, it should compile after this